ha/core/system_health/system_health_metadata.py

```diff
@@ -105,6 +105,7 @@ class SystemHealthHierarchy:
     _server_service = [const.COMPONENTS.SERVER_SERVICE.value, const.COMPONENTS.AGG_SERVICE.value] + _server
     _server_hw = [const.COMPONENTS.SERVER_HARDWARE.value] + _server
     _storage = [const.COMPONENTS.STORAGE_COMPONENT.value,  const.COMPONENTS.STORAGE.value] + _common_hierarchy
+    _cvg = [const.COMPONENTS.CVG.value]
```

Comment:
is the + _common_hierarchy used for hierarchical updates only ? or is there some other purpose also ?

Reply:
Yes,  _common_hierarchy is used only for hierarchical updates.